### ISSUE TITLE
Username already taken error, when registering a user

### ISSUE THREAD

**Issue by JRadl** [strapi/strapi#4463]
**Describe the bug** 
I'm trying to add a new user to my strapi project. When I do I always get an error, even though there are no users with the username in the database.

**Steps to reproduce the behavior**
1. Go to API documentation
2. Select /auth/local/register
3. Click "Try it out"
4. Click "Execute"

**Expected behavior** 
- a new user is created

**Screenshot**

![image](https://user-images.githubusercontent.com/23745826/68293907-32c02d00-008f-11ea-80fa-6e129fdc4751.png)

**System**
- Node.js version: v10.15.3
- NPM version: 6.4.1.
- Strapi version:  3.0.0-beta.17.4
- Database: SQLite
- Operating system: Windows 10

**Additional context** 
- User can be created through the admin panel

**Comment by JackMoseley2001** [comment]
Also experiencing this problem. I also wanted to remove the need to use a username (Using Issue #3757) however continuing to experience 'Username already taken' error

**Comment by lauriejim** [comment]
Hello! I just tried and I had no issue.
Here is the [video record](https://www.loom.com/share/e275e057b3fe48bb900b21d974bc986c) but maybe I miss something 🤔

**Comment by JRadl** [comment]
I have no Idea, how this works, but I played around with it a little bit and after I deleted a relation to another content type from the User content type, it started working. And after I added it back, it still does work. 
This has happend to me once some time ago with the relations... I added a relation to a content type and the server refused to start after that. 
I guess the relations are kinda broken in some way.
